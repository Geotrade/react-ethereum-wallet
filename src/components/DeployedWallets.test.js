import React from 'react';
import renderer from 'react-test-renderer';

describe('wallet html', () => {
  it('renders wallet description', () => {
    const tree = renderer
      .create(
        <React.Fragment>
          <h2>Wallet Contracts</h2>
          <p>
            These contracts are stored on the blockchain and can hold and secure
            Ether. They can have multiple accounts as owners and keep a full log
            of all transactions.
          </p>
          <div className="dapp-clear-fix" />
        </React.Fragment>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  // it('renders wallet link', () => {
  //   const tree = renderer
  //     .create(
  //       <StaticRouter location="someLocation">
  //         <Link
  //           to={{ pathname: '/wallet/new' }}
  //           className="wallet-box create add-contract"
  //         >
  //           <div className="account-pattern">+</div>
  //           <h3>ADD WALLET CONTRACT</h3>
  //         </Link>
  //         <div className="dapp-clear-fix" />
  //       </StaticRouter>
  //     )
  //     .toJSON();
  //   expect(tree).toMatchSnapshot();
  // });
});
