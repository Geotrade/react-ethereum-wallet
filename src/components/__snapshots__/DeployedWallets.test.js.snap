// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wallet html renders wallet description 1`] = `
Array [
  <h2>
    Wallet Contracts
</h2>,
  <p>
    These contracts are stored on the blockchain and can hold and secure Ether. They can have multiple accounts as owners and keep a full log of all transactions.
</p>,
  <div
    className="dapp-clear-fix"
/>,
]
`;
